measurements/api/measurements.py

```diff
@@ -323,67 +332,71 @@ def list_measurements(
     iter_start_time = time.time()
     results = []
 
-    try:
-        for row in q:
-            measurement_id = '{}-{}'.format(MSM_ID_PREFIX, row.msm_no)
-            url = urljoin(
+    with configure_scope() as scope:
+        scope.set_extra("sql_query", str(q))
```

Comment:
This will log next to the error the `sql_query` which triggered the error, to make debugging easier. See: https://docs.sentry.io/enriching-error-data/context/?platform=python